cmd/root.go

```diff
@@ -1,61 +1,363 @@
 package cmd
 
 import (
-	"strings"
-
-	"os"
-
+	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"net/url"
+	"os"
+	"strings"
+	"time"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/andygrunwald/go-jira"
 	"github.com/fsnotify/fsnotify"
+	"github.com/google/go-github/github"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"golang.org/x/oauth2"
 )
 
 var (
-	log             *logrus.Logger
+	log             *logrus.Entry
 	defaultLogLevel = logrus.InfoLevel
 	rootCmdFile     string
 	rootCmdCfg      *viper.Viper
+
+	since               time.Time // The earliest GitHub issue updates we want to retrieve
```

Comment:
Also, by convention we document types and variables by commenting above the variable: https://blog.golang.org/godoc-documenting-go-code